app-crypt/tpm2-openssl/tpm2-openssl-1.2.0.ebuild

```diff
@@ -29,77 +29,18 @@ BDEPEND="
 	virtual/pkgconfig"
 
 PATCHES=(
+	"${FILESDIR}/${PN}-1.2.0-tests-run-with-simulator-in-container.patch"
+	"${FILESDIR}/${PN}-1.2.0-Makefile-add-run-with-simulator-to-extra-dists.patch"
 	"${FILESDIR}/${PN}-1.1.1-build-Fix-undefined-references-when-using-slibtool.patch"
 )
 
 src_prepare() {
-	eautoreconf
 	default
+	eautoreconf
 }
 
 src_test() {
-	dbus_run() {
```

Comment:
The script IS merged upstream. https://github.com/tpm2-software/tpm2-openssl/commit/14daa75f3a1528e2d21252de67cd5f110665f8ae and https://github.com/tpm2-software/tpm2-openssl/commit/ef2d9a53bcdc294316f10f8e6a8e62b4cb688bd7. It didn't get included because an omission in the Makefile.am: https://github.com/tpm2-software/tpm2-openssl/commit/be35c35ace48647bb73ae5028db7e8caaccd038d